### DISCUSSION TITLE
Regarding Table creation

### DISCUSSION THREAD

**Discussion by MGW-SumathyV** [nocodb/nocodb#3334] [Features]
Hi,

 I wanted to create a Global table such that all projects can access the table.
 Is it possible create it? If so how to achieve the same

Thanks and Regards,
Sumathy Venugopal

**Comment by o1lab** [comment]
Could you expand more on the usecase.

Currently this feature is not possible. We can think of it going forward.

**Reply by o1lab** [reply]
Changed to feature discussion
